docs/developer-guide/nodejs-api.md

```diff
@@ -205,6 +205,25 @@ The second parameter `options` is omittable.
 * (`Promise<LintResult[]>`)<br>
   The promise that will be fulfilled with an array of [LintResult] objects. This is an array (despite there being only one lint result) in order to keep the interfaces between this and the [`eslint.lintFiles()`][eslint-lintfiles] method similar.
 
+### ◆ eslint.getRulesMetaForResults(results)
+
+```js
+const results = await eslint.lintFiles(patterns);
+const rulesMeta = await eslint.getRulesMetaForResults(results);
```

Comment:
Is this unnecessary `await` in the example, or was `getRulesMetaForResults` supposed to be async?

Reply:
Unnecessary. Copy-paste mistake.